domains/certificates/Query.API/API/Query.API/ApiModels/Certificate.cs

```diff
@@ -24,4 +24,15 @@ public class Certificate
     /// </summary>
     [JsonPropertyName("gsrn")]
     public string GSRN { get; set; } = "";
+
```

Comment:
After talking with Mads, we also need to add a certificate ID. Use string for now.